packages/api/src/core/translator/dbt.ts

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash'
+import { InvalidArgumentError } from '../../error/error'
+import { Block } from '../block'
+
+/**
+ * This is the default translator, its matching priority is the lowest. It will match all that is not matched by other Translator
+ */
+function match(block: Block): boolean {
+  // TODO: support custom block type
+  return (block.getType() as string) === 'dbt'
+}
+
+/**
+ * Assist param to an executable statement
```

Comment:
more explanation right here?
e.g.
> convert the question-ish block into proper content (table name / injected SQL snippets) in the transclusion

BTW, since the transclusion was assembled as CTE, make sure that the translated subquery is executable.

@TalkWIthKeyboard  Help us to check this plz